web/src/app/app-routing.module.ts

```diff
@@ -55,7 +55,7 @@ const appRoutes: Routes = [
   }
 ];
 @NgModule({
-  imports: [RouterModule.forRoot(appRoutes, { useHash: true })],
+  imports: [RouterModule.forRoot(appRoutes, { useHash: true, relativeLinkResolution: 'legacy' })],
```

Comment:
since routing from Angular 11 is defaulted to ./ for same level instead of ../, would it be better to remove legacy and refactor other link reference ../ to ./ ? So that way it would be future proof